tests/foreman/ui/test_errata.py

```diff
@@ -726,14 +725,23 @@ def test_host_content_errata_tab_pagination(
 
 @pytest.mark.tier2
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_list(session, function_org_with_parameter, lce, target_sat):
+def test_positive_list(
+    function_org_with_parameter,
+    module_org,
+    module_lce,
+    target_sat,
+    session,
+    lce,
+):
     """View all errata in an Org
 
     :id: 71c7a054-a644-4c1e-b304-6bc34ea143f4
 
-    :Setup: Errata synced on satellite server.
+    :Setup:
+        Errata from two custom repos synced on satellite server.
+        Each repository belonging to its own organization.
 
-    :steps: Create two Orgs each having a product synced which contains errata.
+    :steps: Create two Orgs each having a product and repo synced which contains errata.
```

Comment:
`Create two Orgs each having a product and repo synced which contains errata.` This will part of `Setup` section.
Could you please update the `Steps` section as per below code bahaviour?
```suggestion
    :steps:
        1. Setup function scope org with custome repo and sync
        2. Setup module scope org with custome repo and sync
        3. ...<rest of verification steps could be here as per code behaviour>
```